test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -223,14 +241,7 @@ private enum RemoveType
 
     private Command<State<S>, Void, ?> removeNodeAssassinate(RandomSource rs, State<S> state)
     {
-        //TODO (correctness): assassinate CMS member isn't allowed
-        IntHashSet up = asSet(state.topologyHistory.up());
```

Comment:
this was fixed by Marcus on trunk after this logic was written, so cleaning it up